src/components/TableBody.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class TableBody extends React.Component {
+	constructor(props){
+		super(props);
+
+	}
+	render() {
+		const data = this.props.fetched;
```

Comment:
I'm not a huge fan of the `fetched` prop name here. I would go with something like `bookings`. Or if you're trying to make `TableBody` more generic maybe `items` would be better.

Reply:
agree, done